pacu.py

```diff
@@ -1148,6 +1170,8 @@ def display_command_help(self, command_name):
             print('\n    swap_keys\n        Change the currently active AWS key to another key that has previously been set for this session\n')
         elif command_name == 'exit' or command_name == 'quit':
             print('\n    exit/quit\n        Exit Pacu\n')
+        elif command_name == 'load_commands_file':
+            print('\n    load_commands_file\n <commands_file>        Load an existing file with a set of commands to execute')
```

Comment:
Should this line be
```
print('\n    load_commands_file <commands_file>\n        Load an existing file with a set of commands to execute')
```
instead? That way the "<commands_file>" stays on the same line as the load_commands_file

Reply:
Hi @SpenGietz 
Yes, you are right, it should be the way you have described it. 
Would you like me to make this change and push the commit into this PR again?
Thanks.

Reply:
Sure that would be great!

Reply:
Ok - I have pushed a new commit to fix the newline in help message. Could we merge this PR?
Thanks.